packages/asc-ui/src/components/TextArea/TextAreaStyle.ts

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components'
+import { themeColor, focusStyleOutline, themeSpacing } from '../../utils'
+
+type StyleProps = {
+  error?: string | boolean
```

Comment:
This should always be a boolean, never a string.

Reply:
Done